authorization/invitation/service/invitation_service.go

```diff
@@ -381,76 +382,95 @@ func (s *invitationServiceImpl) Accept(ctx context.Context, token uuid.UUID) (st
 		return "", redirectOnFailure, autherrors.NewUnauthorizedError("user deprovisioned")
 	}
 
-	invitationID := inv.InvitationID
-	// If this invitation is for an identity
-	if inv.InviteTo != nil {
-		inviteTo := *inv.InviteTo
-		inviteToIdentity, err := s.Repositories().Identities().Load(ctx, inviteTo)
-		if err != nil {
-			return "", redirectOnFailure, errs.Wrapf(err, "failed to load identity for inviteTo %d", inviteTo)
-		}
+	var resourceID string
 
-		// If the invitation is for a membership, add a membership record
-		if inv.Member {
-			s.Repositories().Identities().AddMember(ctx, inviteToIdentity.ID, currentIdentityID)
-		}
+	err = s.ExecuteInTransaction(func() error {
```

Comment:
the change here is because of a lack of transaction demarcation, isn't it ? Shouldn't it be part of another PR?

Reply:
If it was a significant change I would agree, however I think that including a [rider](https://en.wikipedia.org/wiki/Rider_(legislation)) like this in unrelated PR is fine if it is something minor and doesn't benefit from the overhead of creating another issue, PR and review.  I would hate for our agility to suffer due to overbearing process, especially when something like this is already subject to peer review anyway.

Reply:
ok, fair enough (and thanks for the link to explain the `rider`)